notebooks/notebooks/iclr2022/win-rate-vs-visits.ipynb

```diff
@@ -20,6 +20,7 @@
     "import matplotlib.pyplot as plt\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.            gb.adv_win.sum(), gb.adv_win.count(), alpha=alpha
</pre><p>Why do we use <code>method="beta"</code> above but <code>method="normal"</code> (default) her?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Woops just a typo. Changed to beta everywhere.</p>